dashboards/lodestar_debug_gossipsub.json

```
@@ -92,6 +45,10 @@
   "panels": [
     {
       "collapsed": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "prometheus_local"
```

need to use `${DS_PROMETHEUS}` here, you can just run `node scripts/lint-grafana-dashboards.mjs ./dashboards` to fix the dashboard

```suggestion
        "uid": "${DS_PROMETHEUS}"
```